collectors/InventoryCollector.py

```diff
@@ -13,6 +13,9 @@ def __init__(self):
         self.name = self.__class__.__name__
         self.wait_for_inventory_data()
 
+    def describe(self):
+        self.collect()
+
```

Comment:
This collector needs his own describe() method.

Reply:
Funny thing is, this isn't even be needed. If `describe` is missing, it will automatically call `collect()` at register time.

Reply:
In normal cases, yes, I know. But in this case the collector inherits the method of the BaseCollector. And this is not appropriate.